src/starknet-wrappers.ts

```diff
@@ -199,6 +248,10 @@ export class DockerWrapper extends StarknetWrapper {
             [options.abi]: options.abi
         };
 
+        if (options.accountDir) {
+            addPaths(binds, options.accountDir);
```

Comment:
Why not just directly populate the `binds` object? The `addPaths` function is for the cairo-path option of the compile task (I should've better commented that one).